src/granite_orm/table.cr

```diff
@@ -45,5 +45,13 @@ module Granite::ORM::Table
     def self.primary_name
       @@primary_name
     end
+
+    def self.quoted_table_name
+      @@adapter.quote_table_name(table_name)
+    end
```

Comment:
Do you think there is any drawback to making `#table_name` by default quoted? cc @drujensen